docs/notebooks/unfinished/example_measurement_uhfli.py

```diff
@@ -0,0 +1,77 @@
+import os
+from itertools import cycle
+
+import matplotlib.pyplot as plt
+from qcodes import Station
+
+from qtt.measurements.acquisition import UhfliScopeReader, load_configuration
+
+
+# FUNCTIONS FOR PLOTTING
+
+color_cycler = cycle('bgrcmk')
+
+
+def plot_1D_dataset(plot, records, label_x, label_y):
+    x_data = records[0]
+    y_data_list = records[1:]
+
+    plot.clf()
+    for y_data in y_data_list:
+        plot.plot(x_data.flatten(), y_data.flatten(), color=next(color_cycler), label=y_data.name)
+
+    plot.xlabel(label_x)
+    plot.ylabel(label_y)
+    plot.legend(loc='upper right')
+    plot.draw()
+    plot.pause(0.001)
+
+
+# CREATE THE SCOPE READER AND STATION
+
+device_id = 'dev2338'
+scope_reader = UhfliScopeReader(device_id)
+uhfli = scope_reader.adapter.instrument
+station = Station(uhfli, update_snapshot=False)
+
+
+# INITIALIZE THE SCOPE READER
+
+file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'uhfli.dat')
+configuration = load_configuration(file_path)
+scope_reader.initialize(configuration)
+
+
+# PREPARE THE SCOPE FOR READOUT
+
+scope_reader.number_of_averages = 1
+scope_reader.input_range = (0.5, 1.0)
+scope_reader.sample_rate = 450e6
+scope_reader.period = 1e-4
```

Comment:
If I set the sampling rate before setting the period, everything works as expected. In this example I get back a signal that is 45000 samples long, sampled at 450 MHz so 100us in time. If I change the order and set the period first, then the sample rate I get 180000 samples back or 400us on a time scale.